library/std/src/time.rs

```diff
@@ -359,9 +344,9 @@ impl Instant {
     ///
     /// # Panics
     ///
-    /// This function may panic if the current time is earlier than this
-    /// instant, which is something that can happen if an `Instant` is
-    /// produced synthetically.
+    /// In debug mode this function will panic if the current time is earlier than `self`,
+    /// which is something that can happen if an `Instant` is produced synthetically or due to
+    /// hardware bugs. In release mode it will saturate instead.
```

Comment:
Is this comment still accurate? Based on my read of the issue this should saturate on debug too now

Reply:
Updated the comments with the requested "may panic" changes.